source/templates/partials/_categories.hbs

```diff
@@ -1,6 +1,6 @@
 <ul class="list list--inline categories">
-  <li class="list__item category__item is-active" data-filters="all"><a href="#">all</a></li>
+  <li class="list__item category__item is-active" id="filters-all" data-filters="all"><a href="#">all</a></li>
```

Comment:
Rather than adding IDs or class names to test against, Cypress recommends using a `data-id` attribute when possible. For reference, here's a link to Cypress's [recommended "best practices" for selectors](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements).

Reply:
nice!